bazel/ray.bzl

```diff
@@ -169,3 +170,19 @@ def native_java_library(module_name, name, native_library_name):
         }),
         visibility = ["//visibility:public"],
     )
+
+def cc_library(name, **kwargs):
```

Comment:
maybe rename this to `ray_cc_library`? it is going to be fairly confusing to overwrite builtin rules. if people forget to import `ray.bzl`, it will auto fallback to the default.